generate/generate.go

```diff
@@ -54,17 +54,8 @@ func New(os string) (generator Generator, err error) {
 				"cmd",
 			},
 			Cwd: `C:\`,
-			ConsoleSize: &rspec.Box{
-				Width:  80,
-				Height: 20,
-			},
-		}
-		config.Windows = &rspec.Windows{
-			IgnoreFlushesDuringBoot: true,
-			Network: &rspec.WindowsNetwork{
-				AllowUnqualifiedDNSQuery: true,
```

Comment:
You should add under `SetWindowsNetwork` a `SetWindowsNetworkAllowAllowUnqualifiedDNSQuery` to make this option an easy update for callers when the default changes.

Reply:
That's added now, @jterry75